app/src/main/res/layout/item_reward.xml

```diff
@@ -43,25 +43,46 @@
           style="@style/Caption1Secondary"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
-          android:textColor="@color/ksr_green_700"
+          android:textColor="@color/ksr_green_500"
           tools:text="About $15 USD" />
 
         <TextView
           android:id="@+id/horizontal_reward_title_text_view"
-          style="@style/Headline"
-          android:layout_width="match_parent"
-          android:layout_height="wrap_content"
+          style="@style/RewardTitle"
           android:layout_marginTop="@dimen/activity_vertical_margin"
-          android:textSize="@dimen/title_1"
           tools:text="Beep" />
 
+        <LinearLayout
+          android:id="@+id/horizontal_rewards_item_section"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginTop="@dimen/grid_3"
+          android:orientation="vertical">
+
+          <TextView
+            style="@style/RewardSectionTitle"
+            android:text="@string/rewards_info_includes" />
+
+          <androidx.recyclerview.widget.RecyclerView
```

Comment:
When there's a really long list, it's almost impossible to scroll to the bottom of the reward. Nesting recyclerViews feels really off to me but here we are. Here's an example of what I mean:
![device-2019-04-16-113028 2019-04-16 11_34_47](https://user-images.githubusercontent.com/1289295/56223400-ab03ca00-603b-11e9-9dd5-f665ee9347a4.gif)